src/SFA.DAS.Commitments.Api/Orchestrators/ApprenticeshipOrchestrator.cs

```diff
@@ -21,7 +20,7 @@
 
 namespace SFA.DAS.Commitments.Api.Orchestrators
 {
-    public class ApprenticeshipsOrchestrator
+    public class ApprenticeshipsOrchestrator : IApprenticeshipsOrchestrator
```

Comment:
Changes like this are fine, but bloat the PR a bit so perhaps worth doing on a separate branch.

Reply:
I added these in to facilitate simpler unit tests for the change.